src/typeDefs/schema.graphql

```diff
@@ -49,22 +45,14 @@ type PersonName {
 }
 
 input SpecialtyInput {
-  names: [SpecialtyNameInput]
+  nameEn: String
+  nameJa: String
 }
 
 type Specialty {
   id: ID!
-  names: [SpecialtyName]
```

Comment:
I should change this back... 🤔 we'll definitely want Specialties to have many translations 👀